### ISSUE TITLE
Add new statistics view

### ISSUE THREAD

**Issue by andreaskoepf** [LAION-AI/Open-Assistant#1528]
We currently share statistics about the dataset on the OA discord server but it should be directly integrated in the web app. 

I have added the `/api/v1/stats/cached` and `/api/v1/stats/cached/{name}` backend endpoints with PR https://github.com/LAION-AI/Open-Assistant/pull/1464 . The cached stats are currently recalculated every 60 minutes.

Available stats are currently:
```
    "human_messages_by_lang"
    "human_messages_by_role"
    "message_trees_by_state"
    "message_trees_states_by_lang"
    "users_accepted_tos"
```

We need now a nice front-end page to display the stats. A simple option would be to add a new "Statistics" entry to the command bar on the left side below the "Leaderboard" entry.

**Comment by rjmacarthy** [comment]
I'm willing to cook something up for this.

**Comment by olliestanley** [comment]
Closing as it looks like this was resolved by #1576
